doc/user/content/ingest-data/postgres/amazon-rds.md

```diff
@@ -97,7 +98,11 @@ As a first step, you need to make sure logical replication is enabled.
 
 {{% postgres-direct/create-a-publication-aws %}}
 
-## B. Configure network security
+## B. (Optional) Configure network security
```

Comment:
Oh ... if you change the section heading, then you'll want to check if anyone has previously linked to that heading, because the link won't be `#b-configure-network-security` any longer.  If you run `./ci/test/lint-docs.sh`, it'll highlight for you.